server/aws-lsp-codewhisperer/src/shared/amazonQServiceManager/errors.ts

```diff
@@ -63,3 +63,10 @@ export class AmazonQServiceNoProfileSupportError extends AmazonQError {
         this.name = 'AmazonQServiceNoProfileSupportError'
     }
 }
+
+export class AmazonQServiceAuthenticationExpiredError extends AmazonQError {
+    constructor(message: string = 'Current authentication token is expired.') {
```

Comment:
What do you think about `E_AMAZON_Q_INVALID_CONNECTION`? to match L27
A similar change for the class name as well `AmazonQServiceInvalidConnectionError`